[LibSquoosh] Wrong main entry

https://github.com/GoogleChromeLabs/squoosh/blob/dev/libsquoosh/package.json#L6
Should be `build/index.js`. 

Error on Windows.
```
Error: Cannot find module 'E:\build\index.js'. Please verify that the package.json has a valid "main" entry
```

--- a/libsquoosh/package.json
+++ b/libsquoosh/package.json
@@ -3,7 +3,7 @@
   "version": "0.2.3",
   "description": "A Node library for Squoosh",
   "public": true,
-  "main": "/build/index.js",
+  "main": "./build/index.js",
   "files": [
     "/build/*"
   ],

